var/spack/repos/builtin/packages/rocm-tensile/package.py

```diff
@@ -33,29 +36,48 @@ class RocmTensile(CMakePackage):
         depends_on('rocm-smi@' + ver, type='build', when='@' + ver)
         depends_on('rocm-smi-lib@' + ver, type='build', when='@' + ver)
         depends_on('llvm-amdgpu@' + ver, type='build', when='@' + ver)
-
     # This is the default library format since 3.7.0
     depends_on('msgpack-c@3:', when='@3.7:')
     depends_on('boost@1.58.0', type=('build', 'link'))
+    depends_on('llvm-openmp', type=('build', 'link'))
```

Comment:
this is not the way to do it, the compiler should provide the omp libs if they are strictly necessary, they are currently not link dependencies like here.

Reply:
I will remove the link dependency with a new patch.